src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/*
+ * 
+ *Parent class of address, email and phone. 
+ *Guarantees: immutable; is valid as declared in the subclasses
+ */
+
+public class Contact {
+	
+		
+	public final String value;
+	private boolean isPrivate;
+
+	/*
+	 * Constructor for subclasses name values
+	 * 
+	 */
+	public Contact ( String value, boolean isPrivate ){
+		
+		this.value = value;
+		this.isPrivate = isPrivate;
+	}
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.